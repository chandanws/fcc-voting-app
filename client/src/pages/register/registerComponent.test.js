import React from "react";
import RegisterComponent from "./registerComponent";
import { shallow } from "enzyme";

describe("<RegisterComponent />", () => {
  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<RegisterComponent />);
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  describe("handleChange", () => {});

  describe("handleSubmit", () => {});
});
