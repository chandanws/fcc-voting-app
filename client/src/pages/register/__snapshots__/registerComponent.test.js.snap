// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterComponent /> should render correctly 1`] = `
<div>
  <Typography
    component="h1"
    text="Login"
  />
  <Form
    onSubmit={[Function]}
  >
    <Component
      component="input"
      id={1}
      label="username"
      onChange={[Function]}
      type="text"
      value=""
    />
    <Component
      component="input"
      id={2}
      label="password"
      onChange={[Function]}
      type="password"
      value=""
    />
    <Component
      component="input"
      id={3}
      label="password"
      onChange={[Function]}
      type="password"
      value=""
    />
  </Form>
</div>
`;
